### DISCUSSION TITLE
Dose skywalking-client-js support Vue3

### DISCUSSION THREAD

**Discussion by baixjian** [apache/skywalking#10333] [Q&A]
In this line [L34-L35](https://github1s.com/apache/skywalking-client-js/blob/HEAD/src/errors/vue.ts#L34-L35) use vue2 grammar, What should I do to adapt to vue3? I'd be happy if you could answer.

**Comment by wu-sheng** [comment]
Ping @Fine0830

**Comment by Fine0830** [comment]
Here is the [doc](https://github.com/apache/skywalking-client-js#catching-errors-in-frames-including-react-angular-vue).
